src/condor_starter.V6.1/starter_v61_main.cpp

```diff
@@ -149,6 +149,8 @@ printClassAd( void )
 	// Docker "universe."
 	if( DockerProc::Detect() ) {
 		printf( "%s = True\n", ATTR_HAS_DOCKER );
+		printf( "%s = True\n", ATTR_HAS_DOCKER_REPO);
```

Comment:
What does `HasDockerRepo` actually documented to mean?  IIRC, `DockerProce::Detect()` deliberately doesn't check to see if it has access to "the" Docker repository.

Reply:
There's an assumption here that if we have a working docker binary, that it can access docker repos.  Not sure of an easy way to check this, and I really don't want to hit the docker repo on every startd start.

Reply:
Detecting that a startd can access a docker repo does seem like a hard problem, so I'm wondering if we should just not advertise the ability, instead.  As coded right now, it doesn't add any new information, and seems to make promises we don't have any idea how to keep.

Reply:
The hope is in the future, we can maybe use singularity or other container runtimes to fetch images from docker-style repos.

Reply:
After a conversation with GregT, we've agreed to rename this capability to "HasDockerURL" in hopes to avoid having to deal with *which* repo and/or determining it that repo is actually working.  Instead, we're just saying that we know what to do with `docker://` specifiers.